docs/architecture/state.md

```diff
@@ -1,15 +1,20 @@
-# State
-The state of the Miden rollup describes the current condition of all accounts and note states. It describes what is currently the case. With its state model, using concurrent off-chain state, Polygon Miden aims to realise private, and parallel transaction execution and state bloat minimization. Midens aims to realise:
+Miden rollup state describes the current condition of all accounts and notes; i.e. what is currently the case. 
 
-* **Notes and nullifiers** ensure privacy of note consumption
-* **Flexible data storage** for users who can store their data off-chain or with the network
-* **Parallel transactions** executed concurrently by distinct actors
-* **Concurrent state model** allows block production without knowing the full state
+As the state model uses concurrent off-chain state, Polygon Miden aims for private, and parallel transaction execution and state bloat minimization. 
 
-Privacy is realised from a UTXO-like state model consisting of notes and nullifiers combined with off-chain execution using zero-knowledge proofs. State bloat describes the ever growing state stored in blockchain nodes. Polygon Miden addresses this challenges via its state model that enables concurrent off-chain execution and off-chain storage. Simply put, in Miden users can store their own data locally which reduces the burden on the network - integrity is ensured using zero-knowledge.
+Miden's goals include:
+
+* Notes and nullifiers to ensure privacy of note consumption.
+* Flexible data storage for users who can store their data off-chain or with the network.
+* Parallel transactions executed concurrently by distinct actors.
+* Concurrent state model that allows block production without knowing the full state.
+
+Privacy is enforced by a UTXO-like state model consisting of notes and nullifiers combined with off-chain execution using zero-knowledge proofs. State bloat describes the ever growing state stored in blockchain nodes. Polygon Miden addresses this challenge via its state model that enables concurrent off-chain execution and off-chain storage. Simply put, users can store their own data locally which reduces the burden on the network while integrity is ensured using zero-knowledge.
```

Comment:
Can you make two paragraphs for this? The paragraph should explain (in short) how we achieve the goals above. Privacy and parallelism by UTXO-like state and zkProofs. State bloat minimization by our new state model.